### PR TITLE
feature/CD-42: added a new settings screen

### PR DESCRIPTION
**Pull request by Gerasimov94**

### CODE REVIEW

File: src/screens/Settings/containers/SettingsContainer.ts [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
 import {changeTemperatureScale} from '../actions/settingsActions';
 import {getTemperatureScale} from '../selectors/settingsSelectors';
 
 const mapStateToProps = (state: any) => ({
 	temperatureScale: getTemperatureScale(state),
 });
 
-export default connect(mapStateToProps, {changeTemperatureScale})(Settings);
+export default reduxForm({form: 'user', initialValues: {
+	age: 21,
```
Comment (Gerasimov94) [inline src/screens/Settings/containers/SettingsContainer.ts:15]: temporary values

File: src/screens/Settings/components/SettingsPasswordInput.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {Text, KeyboardAvoidingView, TextInput} from 'react-native';
+
+import styles from '../styles/Settings.styles';
+
+export default class SettingsPasswordInput extends React.PureComponent<any> {
```
Comment (CodeAnnihilator) [inline src/screens/Settings/components/SettingsPasswordInput.tsx:6]: any

File: src/common/components/AvatarUpload.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		alignItems: 'center',
+		justifyContent: 'center',
+	},
+	imageBox: {
+		width: 100,
+		height: 100,
+		borderRadius: 50,
+	},
+});
+
```
Comment (alena-andr) [inline src/common/components/AvatarUpload.tsx:21]: transfer styles to a separate file

File: src/screens/Settings/styles/Settings.styles.ts [hunk 4]
```diff
@@ -12,43 +11,43 @@ export default EStyleSheet.create({
...
-		borderRadius: 10,
-		borderWidth: 1,
-		height: 170,
-		marginVertical: 20,
+		borderBottomWidth: 1,
+		borderBottomColor: '#f3f3f3',
+		paddingVertical: 10,
+	},
+	userBoxTitle: {
+		// fontFamily: 'DIN',
```
Comment (alena-andr) [inline src/screens/Settings/styles/Settings.styles.ts:26]: delete comment

**Pull request merged**